index.html

```diff
@@ -0,0 +1,308 @@
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">26</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">27</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">28</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">29</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">30</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+        <div class="card day" style="width: 9rem; height: 9rem;">
+            <div class="card-body">
+                <h5 class="card-title">31</h5>
+                <p class="card-text"></p>
+                <img src="assets/images/add.png" class="card-img add-icon mx-auto" style="display: none;" alt="Add icon">
+                <a href="#" class="stretched-link"></a>
+            </div>
+        </div>
+      </div>
+
+      <!-- footer -->
+
+      <div class="card">
+        <div class="card-body">
+            <blockquote class="blockquote footer-menu mb-0">
+              <p class="footer-link">Home</p>
+              <p class="footer-link">About</p>
+              <p class="footer-link">Help</p>
+            </blockquote>
+        </div>
+        <div class="card-body">
+          <blockquote class="blockquote mb-0">
+            <p class="footer-name">MealMaster</p>
+          </blockquote>
+        </div>
+      </div>
```

Comment:
if you can think about a way to create cards dynamically in jQuery based on the number of days of the current month